src/main/java/hudson/plugins/sonar/action/SonarCacheAction.java

```diff
@@ -37,16 +38,16 @@
   private List<ProjectInformation> lastProjInfo;
 
   public SonarCacheAction() {
-    this.infoByTaskId = new HashMap<>();
-    this.infoByUrl = new HashMap<>();
+    this.infoByTaskId = new HashMap<String, ProjectInformation>();
+    this.infoByUrl = new HashMap<String, ProjectInformation>();
   }
 
   public List<ProjectInformation> get(SQProjectResolver resolver, long lastBuildTime, List<SonarAnalysisAction> analysis) {
     if (lastRequest != null && age(lastRequest) < TimeUnit.SECONDS.toMillis(10)) {
       return lastProjInfo;
     }
 
-    List<ProjectInformation> list = new ArrayList<>(analysis.size());
+    List<ProjectInformation> list = new ArrayList<ProjectInformation>(analysis.size());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2293)